src/bitmessagekivy/tests/test_sent_message.py

```diff
@@ -0,0 +1,101 @@
+from .telenium_process import TeleniumTestProcess
+from .common import skip_screen_checks
+from .common import ordered
+
+test_address = {'receiver': 'BM-2cWmjntZ47WKEUtocrdvs19y5CivpKoi1h'}
```

Comment:
here try the autoresponder

Reply:
Added autoresponder_address as receiver address